crates/secrets-types/src/lib.rs

```diff
@@ -119,7 +118,9 @@ impl Context {
 /// default to retrieving the latest version of the secret.
 #[derive(Serialize, Deserialize)]
 pub struct SecretRequest {
-    // The name of the secret
+    /// An identifier of the secret as stored in the secret store. This could
+    /// be a key, a path, or any other identifier that the secret store uses to
+    /// retrieve the secret.
```

Comment:
```suggestion
    /// An identifier of the secret as stored in the secret store. 
    ///
    /// A `SecretRequest` can be a key, path, or any other identifier that the secret store uses to
    /// retrieve a secret.
```

Reply:
Adding the space here makes the documentation show a little preview properly in cargo doc IIRC